config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  post 'login' => 'auth#authenticate_user'
-  get 'home' => 'auth#home'
+  scope '/api' do
+    post 'login' => 'auth#authenticate_user'
+    get 'check' => 'auth#check'
+  end
+  #  Send static files
+  match '/register' => 'static#base', via: [:get]
```

Comment:
Is this going to be needed for all client routes? If so, we should do something like:
`get '*path' => 'static#base'`